subcommands/config/log.go

```diff
@@ -30,18 +30,52 @@ func doConfigLog(cmd *cobra.Command, args []string) {
 		subcommands.LogConfigs(&subcommands.LogConfigsOptions{
 			Limit: listLimit,
 			ListFunc: func() (*client.DeviceConfigList, error) {
-				return api.FactoryListConfig(factory)
+
+				list, err := api.FactoryListConfig(factory)
+				for _, dc := range list.Configs {
+					if len(dc.CreatedBy) > 0 {
+						user, _ := api.UserAccessDetails(factory, dc.CreatedBy)
+						dc.CreatedBy = user.Name
+					}
+				}
+				return list, err
+			},
+			ListContFunc: func(url string) (*client.DeviceConfigList, error) {
+				list, err := api.FactoryListConfigCont(url)
+				for _, dc := range list.Configs {
+					if len(dc.CreatedBy) > 0 {
+						user, _ := api.UserAccessDetails(factory, dc.CreatedBy)
+						dc.CreatedBy = user.Name
```

Comment:
What about the performance part of this?

IIRC we keep a maximum of 10 config log entries, so it should be _not so bad_ to make 10 serial user API calls.
But, we have plans to increase the config history limit to 20 or 50 in the future, as it is highly likely some customers might ask for that.
That would make performance implications much worse.

So, I think we can be smart here:
- Most of the time, `CreatedBy` will be the same for all config modifications (at most 3 unique values in 99% percentile).
  That allows to make a trivial optimization by grabbing all unique `CreatedBy` values, and only make one request for each of them.
- The above should be enough for now, so I will not propose to parallelize.
  But at least we need to add a comment like _this can be parallelized if config history limit grows above 10_.